internal/storage/migrations/V0002.Webauthn.mysql.up.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE IF NOT EXISTS webauthn_devices (
+    id INTEGER AUTO_INCREMENT,
+    created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    used TIMESTAMP NULL DEFAULT NULL,
+    rpid TEXT,
+    username VARCHAR(100) NOT NULL,
+    description VARCHAR(30) NOT NULL DEFAULT 'Primary',
+    kid VARCHAR(100) NOT NULL,
+    public_key BLOB NOT NULL,
+    attestation_type VARCHAR(32),
+    transport VARCHAR(20) DEFAULT '',
+    aaguid CHAR(36) NOT NULL,
+    sign_count INTEGER DEFAULT 0,
+    clone_warning BOOLEAN NOT NULL DEFAULT FALSE,
+    PRIMARY KEY (id),
+    UNIQUE KEY (username, description),
+    UNIQUE KEY (kid)
+);
+
+INSERT INTO webauthn_devices (id, rpid, username, description, kid, public_key, attestation_type, aaguid, sign_count)
+SELECT id, '', username, description, TO_BASE64(key_handle), public_key, 'fido-u2f', '00000000-0000-0000-0000-000000000000', 0
+FROM u2f_devices;
+
+DROP TABLE IF EXISTS u2f_devices;
```

Comment:
Think this may need a bit of discussion. Essentially the data between the tables is  the same, just webauthn has additional columns to support the FIDO2 fields. Technically the way we upgrade it should be impossible to have issues, I suppose potentially if the upgrade is interrupted.